### ISSUE TITLE
JIRA display issue status fields

### ISSUE THREAD

**Issue by dragoon** [wtfutil/wtf#203]
#### What problem does this solve?
We use the JIRA within the agile methodology, the work in organized in Sprints, and every issue has a status field (Ready --> In Dev --> In Review --> ...): https://confluence.atlassian.com/adminjiraserver071/defining-status-field-values-802592405.html
It would be good to be able to display this field somehow.

#### How do envision it working?
Status (or any other) field could be displayed together with the type of the issue (story, task) and configured through the config file.

**Comment by senorprogrammer** [comment]
Great idea.

**Comment by voldemortensen** [comment]
I spent a fair amount of time trying to implement this. I was able to implement this fairly easily on a specific JIRA project, but abstracting the logic out to work with many board configurations would be quite complex. I'm going to mull it over for a bit to see if I can come up with something.

<img width="1271" alt="image" src="https://user-images.githubusercontent.com/7786383/42770737-6400dac0-88e2-11e8-8d5d-2228d4a4767a.png">
